packages/react-components/src/components/AutoComplete/AutoComplete.mdx

```diff
@@ -0,0 +1,84 @@
+import { Canvas, ArgTypes, Meta, Title } from '@storybook/blocks';
+
+import * as AutoCompleteStories from './AutoComplete.stories';
+
+<Meta of={AutoCompleteStories} />
+
+<Title>AutoComplete</Title>
+
+[Component API](#ComponentAPI) | [Content Spec](#ContentSpec)
+
+<Canvas of={AutoCompleteStories.Default} sourceState="none" />
+
+#### Example implementation
+
+```jsx
+<AutoComplete options={["Option 1", "Option 2", "Option 3"]} placeholder="Placeholder text" onChange={(e) => {}} />
+```
+
+## Component API <a id="ComponentAPI" />
+
+The `AutoComplete` component is a wrapper around the `Input` component that provides a list of suggestions based on the user's Input. It requires a list of unique `options` to be passed. All of `Input`'s props are available and the component can be used both controlled and uncontrolled.
+
+
+<ArgTypes of={AutoCompleteStories.Default} sort="requiredFirst" />
```

Comment:
We keep the information about the API component at the end of the documentation, please change it to be the same as the others docs pages